src/lancie-admin-content.html

```diff
@@ -48,6 +50,28 @@
           notify: true
         }
       },
+
+      ready: function () {
+        // Catch global toast events
+        this.addEventListener('toast', function(e) {
+          this.showToast(e.detail);
+        }.bind(this));
+
+        // Since the paper-dialogs are put here, the toast has to be moved here. If this is not done, the toasts will appear behind the backdrop.
+        window.document.body.appendChild(this.$.globalToast);
```

Comment:
This isn't needed anymore since we are not appending the dialogs to the body anymore. The toast is already the top-most element.